src/parameterizations/vertical/MOM_ALE_sponge.F90

```diff
@@ -663,16 +663,23 @@ subroutine init_ALE_sponge_diags(Time, G, diag, CS, US)
   type(ALE_sponge_CS),     intent(inout) :: CS   !< ALE sponge control structure
   type(unit_scale_type),   intent(in)    :: US   !< A dimensional unit scaling type
   ! Local Variables
+  character(len=:), allocatable  :: tend_unit  ! The units for a sponge tendency diagnostic.
+  real :: tend_conv  ! The conversion factor use for the sponge tendency [A T-1 ~> a s-1]
   integer :: m
 
   CS%diag => diag
 
   do m=1,CS%fldno
     CS%id_sp_tendency(m) = -1
-    CS%id_sp_tendency(m) = register_diag_field('ocean_model', &
-      'sp_tendency_' // CS%Ref_val(m)%name, diag%axesTL, Time, &
-      'Time tendency due to restoring ' // CS%Ref_val(m)%long_name, &
-      CS%Ref_val(m)%unit, conversion=US%s_to_T)
+    if (trim(CS%Ref_val(m)%unit) == 'none') then
```

Comment:
Could the empty string also be used as a "none" unit?  If so, this could be changed to
```
if (trim(CS%Ref_val(m)%unit) == 'none' .or. len_trim(CS%Ref_val(m)%unit) == 0)
```

Reply:
Yes, that would work better by not adding an unnecessary space in the units string.

Reply:
This block of code has now been updated following this suggestion.